### PR TITLE
treehouses themed login message (fixes #577)

### LINKED ISSUE #577: SSH Login Message or message board
Desgin a message board to prompt user about tree-houses image. This message will appear when user ssh to the raspberry pi.

```
Linux li234-205 3.6.5-x86_64-linode28 treehouses/cli#1 SMP Sun Nov 4 12:40:43 EST 2012 x86_64
Last login: Sat Dec 22 13:38:52 2012 from crazy-hostname

 16:07:17 up  2:36,  1 user,  load average: 0.00, 0.01, 0.05
USER     TTY      FROM              LOGIN@   IDLE   JCPU   PCPU WHAT
vagrant pts/0    97-85-36-82.dhcp 16:07    0.00s  0.16s  0.00s w

Filesystem            Size  Used Avail Use% Mounted on
/dev/hda1              24G  1.1G   22G   5% /

Start by typing treehouses help to begin
```

or 

`treehouses intro`
```

Welcome to the treehouses project!

The Open Learning Exchange is a social benefit and for-purpose organization based in Cambridge, Massachusetts. Our mission is to provide universal quality education using open source materials and technology to overcome educational barrier
For more instruction, please visit.
http://treehouses.io/cli/

To begin usage, type treehouses help
```

### PR DESCRIPTION
**Pull request by xavierh93**
If anybody thinks anything needs to be added to the login message let me know.
Currently it says 
"*Welcome to the treehouses project!

The Open Learning Exchange is a social benefit and for-purpose organization based in Cambridge, Massachusetts.
Our mission is to provide universal quality education using open source materials and technology to overcome educational barriers in third world countries.

For more information, please visit: http://treehouses.io/

To begin using treehouses type \`treehouses help\*"

### CODE REVIEW

**Comment by pattanawadee88** [comment]
I still got this message 
```
The programs included with the Debian GNU/Linux system are free software;
the exact distribution terms for each program are described in the
individual files in /usr/share/doc/*/copyright.

Debian GNU/Linux comes with ABSOLUTELY NO WARRANTY, to the extent
permitted by applicable law.
```

File: scripts.d/16_loginmsg.sh [hunk 1]
```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+motd=/etc/motd
```
Comment (dogi) [inline scripts.d/16_loginmsg.sh]: @xavierh93 this path is a buildmachine path and not from the mounted image

**Comment by CalebProvost** [comment]
```
 _                     _                                       _
| |_  _ __  ___   ___ | |__    ___   _   _  ___   ___  ___    (_)  ___
| __|| '__|/ _ \ / _ \| '_ \  / _ \ | | | |/ __| / _ \/ __|   | | / _ \
| |_ | |  |  __/|  __/| | | || (_) || |_| |\__ \|  __/\__ \ _ | || (_) |
 \__||_|   \___| \___||_| |_| \___/  \__,_||___/ \___||___/(_)|_| \___/

Welcome to the treehouses project!

The Open Learning Exchange is a social benefit and for-purpose organization based in Cambridge, Massachusetts.
Our mission is to provide universal quality education using open source materials and technology to overcome educational barriers in third world countries.

For more information, please visit: http://treehouses.io/

To begin using treehouses type `treehouses help`
```

**Comment by pattanawadee88** [comment]
Screen Shot 2019-11-04 at 3 08 22 PM

**Pull request merged**